src/tests/topp/MzTabExporter_8_output.mzTab

```diff
@@ -0,0 +1,63 @@
+PRT	sp|P15977|MALQ_ECOLI	null	null	null	coli-reference_2019-03-28_plus-iRTs_plus-CONs_Plus-AMM_with-dev-rev	null	null	0.0	null	null	null	0	protein_details
+PRT	sp|P21889|SYD_ECOLI	null	null	null	coli-reference_2019-03-28_plus-iRTs_plus-CONs_Plus-AMM_with-dev-rev	null	null	0.0	null	null	null	0	protein_details
+PRT	sp|P33666|YDBA_ECOLI	null	null	null	coli-reference_2019-03-28_plus-iRTs_plus-CONs_Plus-AMM_with-dev-rev	null	null	0.0	null	null	null	0	protein_details
+PRT	sp|P77503|YCJS_ECOLI	null	null	null	coli-reference_2019-03-28_plus-iRTs_plus-CONs_Plus-AMM_with-dev-rev	null	null	0.0	null	null	null	0	protein_details
+
+PSH	sequence	PSM_ID	accession	unique	database	database_version	search_engine	search_engine_score[1]	modifications	retention_time	charge	exp_mass_to_charge	calc_mass_to_charge	spectra_ref	pre	post	start	end	opt_global_spectrum_reference	opt_global_E-Value	opt_global_Luciphor_global_flr	opt_global_Luciphor_local_flr	opt_global_Luciphor_pep_score	opt_global_XTandem_score	opt_global_b_ions	opt_global_b_score	opt_global_delta	opt_global_hyperscore	opt_global_mass	opt_global_nextscore	opt_global_protein_references	opt_global_q-value_score	opt_global_search_engine_sequence	opt_global_cv_MS:1002217_decoy_peptide	opt_global_y_ions	opt_global_y_score	opt_global_cv_MS:1000889_peptidoform_sequence
+PSM	LSPEELKR	0	sp|P33666|YDBA_ECOLI	1	coli-reference_2019-03-28_plus-iRTs_plus-CONs_Plus-AMM_with-dev-rev	null	[, , XTandem, ]	21.127745000000001	2[MS, MS:1002380, false localization rate, 0.0]-UNIMOD:21	914.674740000000043	3	351.178375244140966	351.177637094270949	ms_run[1]:controllerType=0 controllerNumber=1 scan=4269	K	L	48	55	controllerType=0 controllerNumber=1 scan=4269	2.3	0.0	0.0	21.127745000000001	12.5	1	5.0	2.22e-03	12.5	1051.518350000000055	8.0	unique	0.039325842696629	LS(Phospho)PEELKR	0	3	9.300000000000001	LS(Phospho)PEELKR
+PSM	MKSAMTSSPLR	1	sp|P77503|YCJS_ECOLI	1	coli-reference_2019-03-28_plus-iRTs_plus-CONs_Plus-AMM_with-dev-rev	null	[, , XTandem, ]	3.844931000000001	8[MS, MS:1002380, false localization rate, 1.0]-UNIMOD:21	1257.810359999999946	3	435.868133544921989	430.197826740437677	ms_run[1]:controllerType=0 controllerNumber=1 scan=6225	-	V	1	11	controllerType=0 controllerNumber=1 scan=6225	2.3	1.0	1.0	12.195266	12.6	0	0.0	7.88e-03	12.6	1305.581969999999956	10.1	unique	0.034090909090909	MKSAMTSS(Phospho)PLR	0	3	9.5	MKSAMTSS(Phospho)PLR
+PSM	MKSAMTSSPLR	2	sp|P77503|YCJS_ECOLI	1	coli-reference_2019-03-28_plus-iRTs_plus-CONs_Plus-AMM_with-dev-rev	null	[, , XTandem, ]	3.844931000000001	8[MS, MS:1002380, false localization rate, 1.0]-UNIMOD:21	1257.810359999999946	3	435.868133544921989	430.197826740437677	ms_run[1]:controllerType=0 controllerNumber=1 scan=6225	-	V	1	11	controllerType=0 controllerNumber=1 scan=6225	2.3	1.0	1.0	12.195266	12.6	0	0.0	7.88e-03	12.6	1305.581969999999956	10.1	unique	0.034090909090909	MKSAMTSS(Phospho)PLR	0	3	9.5	MKSAMTSS(Phospho)PLR
```

Comment:
Doesnt work! Uses the same PSM again?????

Reply:
maybe just luciphor issue?

Reply:
found it.